src/Snapshooter.Json/Snapshooter.Json.csproj

```diff
@@ -16,4 +16,8 @@
     <ProjectReference Include="..\Snapshooter\Snapshooter.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
This reference is set in the Package.props file, and does not need to be set in every project seperately